### ISSUE TITLE
WCAG Test

### ISSUE THREAD

**Issue by MissAdelliah** [MissAdelliah/Silje_Reppe_SP2_MAR25FT#2]
For color palette chosen
Include
- Wished palette and changes done acording to WCAG AA
- Final palette passed WGAC AA
- Screenshot of page with color combo

**Comment by MissAdelliah** [comment]
Light yellow/beige goes well with the blue, but the blue crashes with black, added a brighter blue if im ever going to use it with black, doubt it. but then i have a wider WCAG AA friendly colour palette
<img width="1314" height="419" alt="Image" src="https://github.com/user-attachments/assets/95829b85-a0b3-442a-a5e3-c2c67ec47476" />

**Comment by MissAdelliah** [comment]
Colour palette
<img width="335" height="249" alt="Image" src="https://github.com/user-attachments/assets/3b02bad6-fdd2-4d24-a848-403ac2b60f0a" />

<img width="875" height="492" alt="Image" src="https://github.com/user-attachments/assets/9861cfd7-d555-4c45-ab6d-895d8bc70a5f" />

**Comment by MissAdelliah** [comment]
What I have per 10.08 
<img width="470" height="1034" alt="Image" src="https://github.com/user-attachments/assets/aa6df094-a5cd-4892-9bd0-2d59ea36e49c" />
